### ISSUE TITLE
about the test dataset

### ISSUE THREAD

**Issue by Juice113** [sail-sg/CLoT#24]
Thanks for your work!
in the paper, "For training purposes, 95% of the samples are randomly selected to construct the training dataset, while the remaining 5% form the test dataset for validation and analysis."
I wonder which are selected as the test set? I didn't find those test data in the dataset.
looking forward to your reply.

**Comment by ichiroex** [comment]
@zhongshsh  I would also like to know more details about the train/test split labels. I’d greatly appreciate it if you could respond when convenient.
